drivers/overlay/encryption.go

```diff
@@ -66,18 +90,15 @@ func (e *encrMap) String() string {
 		b.WriteString(k)
 		b.WriteString(":")
 		b.WriteString("[")
-		for _, s := range v {
-			b.WriteString(s.String())
-			b.WriteString(",")
-		}
+		b.WriteString(fmt.Sprintf("%v", v))
 		b.WriteString("]")
 
 	}
 	return b.String()
 }
 
 func (d *driver) checkEncryption(nid string, rIP net.IP, vxlanID uint32, isLocal, add bool) error {
-	logrus.Debugf("checkEncryption(%s, %v, %d, %t)", nid[0:7], rIP, vxlanID, isLocal)
+	logrus.Debugf("checkEncryption(%s, %v, %d, %t, %t)", nid[0:7], rIP, vxlanID, isLocal, add)
```

Comment:
This is for debugging. An add==true means a container joined this network and vice-versa, therefore the encryption states (install/remove) must be evaluated.